pkg/apis/agent/v1alpha1/validations.go

```diff
@@ -49,6 +53,41 @@ func checkAtMostOneDeploymentOption(b *Agent) field.ErrorList {
 	return nil
 }
 
+func checkAtMostOneDefaultESRef(b *Agent) field.ErrorList {
+	var found int
+	for _, o := range b.Spec.ElasticsearchRefs {
+		if o.OutputName == "default" {
+			found++
+		}
+	}
+	if found > 1 {
+		return field.ErrorList{
+			field.Forbidden(field.NewPath("spec").Child("elasticsearchRefs"), "only one elasticsearchRef may be named 'default'"),
```

Comment:
nit: the field name that we are complaining about is `outputName`, maybe message could hint that?